lte/gateway/python/scripts/enodebd_cli.py

```diff
@@ -167,8 +167,14 @@ def get_enb_status(client, args):
     _print_str_status_line('GPS Longitude', enb_status.gps_longitude)
     _print_str_status_line('GPS Latitude', enb_status.gps_latitude)
     _print_str_status_line('FSM State', enb_status.fsm_state)
-
-
+    _print_str_status_line('GPS Aititude', enb_status.gps_aititude)
```

Comment:
What is aititude? Is this not altitude?

Reply:
![image](https://user-images.githubusercontent.com/87301024/147812697-e066676f-40d9-46fb-bb26-2f03580c1e6c.png)
modified,thank you!